build/PackageDiffIgnore.xml

```diff
@@ -1874,6 +1874,16 @@
 		</Fields>
 
 		<Properties>
+			<!-- BEGIN templated-parent rework -->
+
+			<!-- DependencyObject (codegen for all descendants) -->
+			<Member fullName=".*DependencyObject .*::TemplatedParent" reason="Not in UWP" isRegex="true" />
+			<Member fullName=".*DependencyProperty .*::TemplatedParentProperty" reason="Not in UWP" isRegex="true" />
```

Comment:
I think there won't be much usage of `TemplatedParentProperty`, so this one is probably okay.